resources/views/layouts/navigation.blade.php

```diff
@@ -15,6 +15,14 @@
                     <x-nav-link :href="route('users.index')" :active="request()->routeIs('users.index')">
                         {{ __('Users') }}
                     </x-nav-link>
+                    @if(auth()->check())
```

Comment:
https://laravel.com/docs/8.x/blade#authentication-directives

Reply:
Thanks for the tip